pkgs/top-level/all-packages.nix

```diff
@@ -25940,6 +25940,8 @@ with pkgs;
 
   home-assistant-cli = callPackage ../servers/home-assistant/cli.nix { };
 
+  home-assistant-companion = callPackage ../servers/home-assistant/companion.nix { };
```

Comment:
Shouldn't we place that outside of the server directory, possibly pkgs-by-name because it is a client side app and not really related to the server as that it is running on another device?